tests/extension/ancillary_tags.phpt

```diff
@@ -39,6 +39,11 @@ key=val
 use function datadog\appsec\testing\add_ancillary_tags;
 
 header('Content-type: application/json');
+header('Content-encoding: foobar');
+header('Content-language: pt_PT');
+header('Content-length: 42');
+header('Content-ignored: 42');
```

Comment:
Maybe add an underscore to verify that it's replaced with a dash as expected.

Reply:
It shouldn't do such replacement. The request headers have that transformation because their key in `$_SERVER` is transformed `foo-bar` -> `HTTP_FOO_BAR`. But here we get the headers as they are output in the response header.